loris/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  app:
+    build: .
+    ports:
+     - "8888:8888"
+    environment:
+      - INFRA_BUCKET=platform-infra
+      - CONFIG_KEY=${CONFIG_KEY}
+      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
+      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
+
+  nginx:
+    image: "nginx_loris:${NGINX_TAG}"
```

Comment:
Can you do default values in docker-compose env variables? So this defaults to "latest"?

Reply:
https://docs.docker.com/compose/env-file/ seems to be the way to do it, which is less than ideal.